web/source/kmwuitoggle.js

```diff
@@ -52,9 +52,20 @@ try {
   ui.doFocus = function(someElement,focusing)
   {
     // This callback must be ignored until UI is initialized, or for touch devices (which can never initialize the UI)
-    if(!ui.initialized) return;
+    if(!ui.initialized) {
+      return;
+    }
     
-    if(window.event) someElement=window.event.srcElement;
+    if(window.event) {
+      someElement=window.event.srcElement;
+    }
+
+    // Hides the Toggle interface if a control has KeymanWeb disabled by API calls.
+    if(keymanweb._ActiveControl != null){
```

Comment:
You can see in keymanweb.js that the call to `ui.controlfocused` event handlers includes an `'activeControl'` property:

````
  keymanweb.doControlFocused = function(_target,_activeControl)
  {
    var p={}; p['target']=_target; p['activeControl']=_activeControl;  
    return util.callEvent('kmw.controlfocused',p);
  }
````

You could do use that in kmwuitoggle.js, e.g. pass that through in line 116 of kmwuitoggle.js as an additional parameter to `ui.doFocus`:

````
keymanweb['addEventListener']('controlfocused',function(params){ui.doFocus(params.target,params.activeControl,true);});
````